test/images/rhel-atomic

```diff
@@ -1 +1 @@
-rhel-atomic-d5e0544247f58ab778219aa786d21ecdc610b7a0.qcow2
\ No newline at end of file
+rhel-atomic-67973815e06227393fd42c227f7afaa82a851a2a.qcow2
```

Comment:
Where did you upload this? In your regular image store there is only a recent upload of the previous image to be found.